content/community/_index.html

```diff
@@ -12,13 +12,13 @@
 
 <h2>Software</h2>
 
-<p>Besides the Haiku repo that provides the packages that make up the operating
+<p>Besides the Haiku repository that provides the packages that make up the operating
 system and HaikuPorts which offers a large number of 3rd party software, there
-are several other repos and software collections that are curated by community
-members.</p>
-<p>The Haiku project isn't legally or otherwise responsible for the contents of those sites.</p>
+are several other repositories and software collections that are independently curated by
+community members.</p>
+<p>Haiku isn't legally or otherwise responsible for the contents of those sites.</p>
```

Comment:
this intentionally said "The Haiku project". "Haiku" isn't a person or legal entity or anything, and makes no sense here.